.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"      # current default Python on Travis CI
+  - "3.7"
+  - "3.8"
+#  - "3.8-dev"  # 3.8 development branch
+#  - "nightly"  # nightly build
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
```

Comment:
you shouldn't have to install the requiements.txt file since tox will install the whole scholia package using `pip install .`.